dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/RegistrationRestRepository.java

```diff
@@ -130,14 +129,14 @@ public RegistrationRest createAndReturn(Context context) {
             try {
                 if (!AuthorizeUtil.authorizeUpdatePassword(context, eperson.getEmail())) {
                     throw new DSpaceBadRequestException("Password cannot be updated for the given EPerson with email: "
-                        + eperson.getEmail());
+                                                            + eperson.getEmail());
                 }
                 accountService.sendForgotPasswordInfo(context, registrationRest.getEmail());
             } catch (SQLException | IOException | MessagingException | AuthorizeException e) {
                 log.error("Something went wrong with sending forgot password info email: "
-                    + registrationRest.getEmail(), e);
+                              + registrationRest.getEmail(), e);
             }
-        } else if (accountType.equalsIgnoreCase(TYPE_REGISTER)) {
+           } else if (accountType.equalsIgnoreCase(TYPE_REGISTER)) {
```

Comment:
Please remove the indentation here before the `else if`. This can go back to how it was